src/Database/LSMTree/Internal/Snapshot/Codec.hs

```diff
@@ -459,13 +459,14 @@ instance DecodeVersioned RunNumber where
 -- SnapIncomingRun
 
 instance Encode (SnapIncomingRun RunNumber) where
-  encode (SnapMergingRun mpfl nr ne uc smrs) =
+  encode (SnapMergingRun mpfl nr ne uc sc smrs) =
        encodeListLen 6
```

Comment:
```suggestion
       encodeListLen 7
```

Reply:
Thanks! That'll probably fix the snapshot error.